phpcs.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<ruleset name="FINDOLOGIC">
+    <file>src</file>
+    <file>tests</file>
+
+    <arg name="colors"/>
+    <arg value="p"/>
+
+    <arg name="report-full"/>
+
+    <!--
+       You can hard-code custom php.ini settings into your custom standard.
+       The following tag sets the memory limit to 128M.
+    -->
+    <ini name="memory_limit" value="128M"/>
+
+    <!--
+       Include all sniffs in the PEAR standard. Note that the
```

Comment:
It's the PSR2 standard, not the PEAR standard. This is probably a copy-paste error.

Reply:
Done.